### PR TITLE
VONK 2744: Add _total to performance search parameters

### PR DESCRIPTION
**Pull request by ylboerner**

### CODE REVIEW

File: performance/performance_firely_server.rst [hunk 1]
```diff
@@ -31,6 +31,11 @@ Search Parameters
 
 When a resource is sent to Firely Server for storage, Firely Server indexes the resource for all the search parameters that are applicable to the resource. The more search parameters are known to Firely Server, the more resources will be used for indexing - both time (for the indexing processing) and storage (for storing the values of each search parameter). This also increases the size of the index tables and indexes, and therefore querying times. Thus, if you know you will only use a portion of the predefined search parameters you can choose to delete the others from the Administration API - see :ref:`conformance` and :ref:`supportedmodel`.
 
+Additionally, these search parameters can be disabled in order to improve performance:
```
Comment (alexzautke) [inline performance/performance_firely_server.rst:34]: I would maybe refrain from using "disabled", the search parameter is still available.
Suggestion: "Additionally, these search parameters can have a large impact on the search performance:"

File: performance/performance_firely_server.rst [hunk 2]
```diff
@@ -31,6 +31,11 @@ Search Parameters
 
 When a resource is sent to Firely Server for storage, Firely Server indexes the resource for all the search parameters that are applicable to the resource. The more search parameters are known to Firely Server, the more resources will be used for indexing - both time (for the indexing processing) and storage (for storing the values of each search parameter). This also increases the size of the index tables and indexes, and therefore querying times. Thus, if you know you will only use a portion of the predefined search parameters you can choose to delete the others from the Administration API - see :ref:`conformance` and :ref:`supportedmodel`.
 
+Additionally, these search parameters can be disabled in order to improve performance: 
+
+#. ``_total``: Searchbundles contain the element ``total`` which indicates the total number of resources that match the query's search parameters. Disabling this element saves the query that would generate the result. Set ``_total = none`` in your HTTP request's parameters to disable it.
```
Comment (alexzautke) [inline performance/performance_firely_server.rst:36]: Suggestion as "query" might be an overloaded term here:

""Search bundles contain the element ``total`` which indicates the total number of resources that match the request's search parameters. Setting this parameter to ``_total = none`` results in faster searches as it saves the query that would generate the result of the aforementioned element."

**Review by alexzautke — CHANGES_REQUESTED** [review]

File: performance/performance_firely_server.rst (hunk 2 above)
Comment (ylboerner) [reply performance/performance_firely_server.rst:36]: Agreed!

File: performance/performance_firely_server.rst (hunk 1 above)
Comment (ylboerner) [reply performance/performance_firely_server.rst:34]: Agreed! I'll refrain from using 'large' as that is dependent on the size of the database but it makes more sense your way.

**Review by alexzautke — APPROVED** [review]

**Pull request merged**